packages/nuxi/src/commands/dev.ts

```diff
@@ -58,7 +58,10 @@ export default defineNuxtCommand({
       showURL: false,
       clipboard: args.clipboard,
       open: args.open || args.o,
-      port: args.port || args.p || process.env.NUXT_PORT || config.devServer.port,
+      port: {
+        port: args.port || args.p || process.env.NUXT_PORT || config.devServer.port,
+        portRange: [3000, 3100]
```

Comment:
I believe we could use it as default in listhen directly to use other fallbacks (actually this was behavior for trying 3000 + a range but behavior regressed as result of another fix)

Reply:
Okay, so let's resolve upstream in `listhen` rather than here. 👍 

![whack-a-mole](https://media.giphy.com/media/ebITvSXYKNvRm/giphy.gif)

Reply:
Agree, it seems to be a regression in `listhen` but thank you very much for the PR @RicardoErii ❤️

Reply:
It's okay, so I close this PR?

Reply:
Let's keep it open until https://github.com/unjs/listhen/issues/62 is fixed